assets/blocks/course-outline/style.scss

```diff
@@ -22,18 +22,21 @@ $spacing: 16px;
 		background: none;
 		border: 0;
 		color: inherit;
-		flex-basis: 4%;
-		margin: 0 0 0 10px;
+		margin: 0 -5px 0 10px;
 		cursor: pointer;
-		height: 24px;
-		font-size: 24px;
+		svg {
+			width: 32px;
+			height: 32px;
+			margin: -12px 0;
+			fill: currentColor;
+		}
 
 		&:hover, &:focus {
 			text-decoration: none;
 		}
 
 		&.collapsed {
-			display: none;
+			transform: scaleY(-1)
```

Comment:
`transition: transform 0.25s;` for a slightly less jarring transition.

Reply:
Tried a few variations but wasn't sure of it. The content collapse duration is `0.5s`, looks a bit off if the arrow is different, but it could work. The scale was just to make sure the up and down arrows are symmetrical, but with animation it could be better as a rotation:

Rotate:
![QuickTime movie](https://user-images.githubusercontent.com/176949/95248649-e4278f80-0817-11eb-9382-44e0046f6f39.gif)

Scale:
![QuickTime movie](https://user-images.githubusercontent.com/176949/95248726-07523f00-0818-11eb-9cc4-3387619ee17b.gif)

Reply:
Yeah, I paired `transition: all 0.25s;` with `transform: rotate(180deg);` in the Collapsible Content plugin, so it may be closer to what users expect.

Reply:
Went with the rotation and tuned timings a bit in a402f2d